src/Cairo.jl

```diff
@@ -98,9 +98,11 @@ export
     # push+pop group
     push_group, pop_group
 
-function write_to_ios_callback(s::Ptr{Void}, buf::Ptr{UInt8}, len::UInt32)
-    n = ccall(:ios_write, UInt, (Ptr{Void}, Ptr{Void}, UInt), s, buf, len)
-    @compat Int32((n == len) ? 0 : 11)
+@static if is_apple()
```

Comment:
actually i wanted to test if `@static` can be used to exclude code from the coverage calculation. I think the compat still applies for <0.5